environs/config/config_test.go

```diff
@@ -832,8 +836,7 @@ func (test configTest) check(c *gc.C) {
 		c.Assert(cfg.ContainerImageStream(), gc.Equals, "released")
 	}
 
-	resourceTags, cfgHasResourceTags := cfg.ResourceTags()
-	c.Assert(cfgHasResourceTags, jc.IsTrue)
+	resourceTags, _ := cfg.ResourceTags()
```

Comment:
I thought this check was included mistakenly, because it was absent from the above minimalConfigAttrs

However, the real reason is because I was using `NoDefaults` instead of `UseDefaults`

Fixing